transforms/dialog-full-screen-app-wrapper/dialog-full-screen-app-wrapper.js

```diff
@@ -0,0 +1,120 @@
+/*
+ * Add <AppWrapper> around the children on <DialogFullScreen>
+ * <DialogFullScreen><AppWrapper>{ children }</ AppWrapper></ DialogFullScreen>
+ */
+import registerMethods from "../registerMethods";
+
+function transformer(fileInfo, api, options) {
+  const j = api.jscodeshift;
+  registerMethods(j);
+
+  const addImport = () => {
+    // DialogFullScreen import
+    const dialogFullScreenImport = root.find(j.ImportDeclaration, {
+      source: {
+        type: "Literal",
+        value: "carbon-react/lib/components/dialog-full-screen"
+      }
+    });
+
+    const appWrapperImportString = "carbon-react/lib/components/app-wrapper";
+
+    // AppWrapper import
+    let appWrapperImport = root.find(j.ImportDeclaration, {
+      source: {
+        type: "Literal",
+        value: appWrapperImportString,
+      },
+    });
+
+    // if no AppWrapper import exists already add one
+    if (!appWrapperImport.size()) {
+      appWrapperImport = j.importDeclaration(
+        [j.importDefaultSpecifier(j.identifier("AppWrapper"))],
+        j.stringLiteral(appWrapperImportString)
+      );
+    
+      // insert AppWrapper import after DialogFullScreen import
+      dialogFullScreenImport.insertAfter(appWrapperImport);
+
+      return true;
+    } else {
+      return false;
+    }
+  };
+
+  const addAppWrapper = (dialogFullScreen, addedImport) => {
+    let appWrapperIdentifier = "AppWrapper";
+
+    // if there already was an import
+    if (!addedImport) {
+      const appWrapperImport = root.find(j.ImportDeclaration, {
+        source: {
+          type: "Literal",
+          value: "carbon-react/lib/components/app-wrapper",
+        },
+      });
+
+      // find identifier used for input
+      appWrapperIdentifier = appWrapperImport.paths()[0].value.specifiers[0].local.name;
```

Comment:
There is a getter we can use for this
```suggestion
      appWrapperIdentifier = appWrapperImport.get('specifiers', 0, 'local', 'name').value
```